storybook/src/components/Button/Button.docs.mdx

```diff
@@ -37,4 +37,13 @@ Use tertiary buttons for unimportant calls to action – as many as necessary.
 
 ### Button with an icon
 
-<Canvas of={ButtonStories.ButtonWithAnIcon} />
+<Canvas of={ButtonStories.WithIcon} />
+
+### Text wrapping
+
+The label can occasionally wrap over multiple lines, for various reasons.
+A shorter text might be unavailable.
+Two buttons could sit next to each other on a narrow screen.
+Or the user has configured a large font on their device.
```

Comment:
Do we want to add that this isn't recommended?

Reply:
I don’t think we can – there’s no way around a wrapping button label if it’s on a phone with a text size zoomed to 300%.

We can update to ‘Keep in mind that the label can occasionally wrap over multiple lines’. We don’t suggest using short labels yet; I can add that to the top of the docs in support?